src/firefly/js/ui/FormPanel.jsx

```diff
@@ -79,7 +79,7 @@ export const FormPanel = function (props) {
     );
 
     return (
-        <Stack component={Sheet} className='ff-FormPanel' spacing={1} p={1} height={1} sx={{...style, ...sx}}>
+        <Stack component={Sheet} className='ff-FormPanel' flexGrow={1} spacing={1} p={1} height={1} sx={{...style, ...sx}}>
```

Comment:
You should not add a `flexGrow` here unless you have confirmed that it did not break anywhere else.  This component it used in many places.  Instead, you can add `flexGrow` to `sx`.

Reply:
> You should not add a `flexGrow` here unless you have confirmed that it did not break anywhere else. This component it used in many places. Instead, you can add `flexGrow` to `sx`.

@loitly I made change as you requested, please take a look 

> You should not add a `flexGrow` here unless you have confirmed that it did not break anywhere else. This component it used in many places. Instead, you can add `flexGrow` to `sx`.

@loitly I reverted change to FormPanel and added change to SearchPanel where is uses FormPanel component with  flexGrow added